toolchain/check/import_cpp.cpp

```diff
@@ -61,7 +62,9 @@ static auto GenerateAst(Context& context, llvm::StringRef importing_file_path,
                                                     diagnostic_options.get());
   // TODO: Share compilation flags with ClangRunner.
   auto ast = clang::tooling::buildASTFromCodeWithArgs(
-      GenerateCppIncludesHeaderCode(context, imports), {},
+      GenerateCppIncludesHeaderCode(context, imports),
+      {// Parse C++ (and not C)
+       "-x", "c++"},
```

Comment:
```suggestion
      // Parse C++ (and not C).
      {"-x", "c++"},
```

Per style: "Only use line comments (with //, not /* ... */), on a line by themselves"
https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md

Reply:
Done.